modules/currency-l0/src/main/scala/org/tessellation/currency/l0/modules/Services.scala

```diff
@@ -81,7 +84,19 @@ object Services {
         maybeDataApplication
       )
 
-      proposalSelect = ProposalOccurrenceSelect.make()
+      getTrusts = Applicative[F].product(
+        p2PClient.l0Trust.getPreviousSeedlistPeerLabels.run(cfg.globalL0Peer),
+        p2PClient.l0Trust.getCurrentTrust.run(cfg.globalL0Peer)
```

Comment:
Move into the following `else` since it's only. used there?

Reply:
Re-factored to

```
      proposalSelect =
        if (cfg.environment === Dev)
          ProposalOccurrenceSelect.make()
        else {
          val getTrusts = Applicative[F].product(
            p2PClient.l0Trust.getPreviousSeedlistPeerLabels.run(cfg.globalL0Peer),
            p2PClient.l0Trust.getCurrentTrust.run(cfg.globalL0Peer)
          )

          val primary = ProposalTrustSelect.make(getTrusts, cfg.proposalSelect)
          val secondary = ProposalOccurrenceSelect.make()

          ProposalSelectWithFallback.make(primary, secondary)
        }
```